### ISSUE TITLE
[bug] Cannot import chatgpt package, es module not supported in cloud function.

### ISSUE THREAD

**Issue by maslow** [labring/laf#785]
### Search before asking

- [X] I had searched in the [issues](https://github.com/labring/laf/issues?q=is%3Aissue) and found no similar issues.

### Environment

preview.laf.run

### What happened

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/7122237/218911368-9b0e2f39-728d-4353-a1e8-3610e01fdf10.png">

The problem such as image above.

### How to reproduce

1. add npm package `chatgpt` to laf app
2. import & use it in cloud function
3. got error logs while requesting it

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by maslow** [comment]
These node codes works well in local node environment. (not laf cloud function).
```js
async function main() {
  const gpt = await import("chatgpt")
  console.log(gpt)
}

main();
```

**Comment by maslow** [comment]
fixed in #819
